app/javascript/components/workers-form/helpers.jsx

```diff
@@ -0,0 +1,192 @@
+/* Transform input value to bytes (12545411) */
+export const toBytes = (value) => {
```

Comment:
js doc would be nice for functions like these. I think it will be reused in more places in future.